LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Hoang Pham Huu <phamhuuhoang@gmail.com>
+Copyright (c) 2010-2016 Curiosity Media, Inc.
```

Comment:
I'm not sure we can just change the copyright like this. Can you spend 15 minutes googling to try to see how copyright works for forks?